app/src/main/java/firebasebarcelona/firepadel/ui/courts/view/CourtListFragment.java

```diff
@@ -141,7 +141,7 @@ public void onAuthStateChanged(@NonNull FirebaseAuth auth) {
         if (user != null) {
           saveLocalPlayer(user);
         } else {
-          Toast.makeText(getActivity(), "Auth fail", Toast.LENGTH_SHORT).show();
+          //TODO show some kind of error
```

Comment:
Yes, I mean, we could use issue to keep track of what we have to do, or projects, or trello, or whatever. But keep the code clean and free of comments and TODO